data/json/daschland.json

```diff
@@ -0,0 +1,300 @@
+{
+    "$schema": "https://raw.githubusercontent.com/dasch-swiss/dsp-meta-svc/main/docs/services/metadata/schema-metadata.json",
```

Comment:
this is the old schema again. Please use the one that all the other files in this repo use.